compose/common.yml

```diff
@@ -1,4 +1,4 @@
-version: '2.1'
+version: "2.0"
```

Comment:
These two versions are almost identical, did you have issues with `2.1`?

Reply:
Yes, there are extra healthcheck options for the `depends_on` terms in `2.1` which we don't support in the `docker-compose.yml` file. I had to drop it to `2.0` to make `docker-compose config` produce a compatible file for `balena push`

Reply:
Wait, I'm confused. You don't *have* to supply healthchecks for `depends_on` in 2.1, is this what you mean? We certainly don't in BoB, so I don't know why we'd go backwards (certainly ideally we want both BoB and OB to be using the same dc version given their parallels).
On that subject, docker-compose creates a single config from multiple compose files so you'll always promote to the highest version required.

Reply:
As discussed, this is due to `docker-compose config...` producing unsupported terms in the resulting `docker-compose.yml` - the simplest way to get a working output is to drop the version down.